nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,167 @@
+        Region definitions, as one image of labels.
+
+    labels : :obj:`list` of :obj:`str`, optional
+        Full labels corresponding to the labels image. This is used
+        to improve reporting quality if provided.
+
+        .. warning::
+            The labels must be consistent with the label
+            values provided through `labels_img`.
+
+    background_label : :obj:`int` or :obj:`float`, optional
+        Label used in labels_img to represent background.
+        Warning: This value must be consistent with label values and
+        image provided.
+        Default=0.
+
+    mask_img : Niimg-like object, optional
+        See :ref:`extracting_data`.
+        Mask to apply to regions before extracting signals.
+    %(smoothing_fwhm)s
+    standardize : {'zscore', 'psc', True, False}, optional
+        Strategy to standardize the signal.
+
+            - 'zscore': the signal is z-scored. Timeseries are shifted
+            to zero mean and scaled to unit variance.
+            - 'psc':  Timeseries are shifted to zero mean value and scaled
+            to percent signal change (as compared to original mean signal).
+            - True : the signal is z-scored. Timeseries are shifted
+            to zero mean and scaled to unit variance.
+            - False : Do not standardize the data.
+
+        Default=False.
+    %(standardize_confounds)s
+    high_variance_confounds : :obj:`bool`, optional
+        If True, high variance confounds are computed on provided image with
+        :func:`nilearn.image.high_variance_confounds` and default parameters
+        and regressed out. Default=False.
+    %(detrend)s
+    %(low_pass)s
+    %(high_pass)s
+    %(t_r)s
+    dtype : {dtype, "auto"}
+        Data type toward which the data should be converted. If "auto", the
+        data will be converted to int32 if dtype is discrete and float32 if it
+        is continuous.
+
+    resampling_target : {"data", "labels", None}, optional.
+        Gives which image gives the final shape/size:
+
+            - "data" means the atlas is resampled to the
+            shape of the data if needed
+            - "labels" means en mask_img and images provided to fit() are
+            resampled to the shape and affine of maps_img
+            - None means no resampling: if shapes and affines do not match, a
+            ValueError is raised
+
+        Default="data".
+    %(memory)s
+    %(memory_level)s
+    %(n_jobs)s
+    %(verbose0)s
+    strategy : :obj:`str`, optional
+        The name of a valid function to reduce the region with.
+        Must be one of: sum, mean, median, minimum, maximum, variance,
+        standard_deviation. Default='mean'.
+
+    reports : :obj:`bool`, optional
+         If set to True, data is saved in order to produce a report.
+         Default=True.
+
+    See also
+    --------
+    nilearn.maskers.NiftiMasker
+    nilearn.maskers.NiftiLabelsMasker
+
+    """
+
+    def __init__(self, *args, n_jobs=1, **kwargs):
```

Comment:
I would prefer to have the full function signature here rather than forwarding *args, **kwargs